docs/_posts/examples/3400-01-27-set-popup.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: example
+category: example
+title: Set a popup to a marker
+description: Set a popup to a marker and display it on click.
+tags:
+  - controls-and-overlays
+---
+<style>
+
+#marker {
+    background-image: url('{{site.baseurl}}/assets/washington-monument.jpg');
+    background-size: cover;
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+    cursor: pointer; }
```

Comment:
nitpicky detail -- can you put the closing bracket on a new line for these 2 css rules?